gradle/findbugs-exclude.xml

```diff
@@ -266,13 +266,6 @@ For a detailed description of findbugs bug categories, see http://findbugs.sourc
 
     <Match>
         <!-- TODO: fix this (see KAFKA-4996) -->
-        <Class name="org.apache.kafka.streams.state.internals.Segments"/>
```

Comment:
The TODO was removed, but there is still one TODO remaining pointing to the same JIRA. I was initially confused as well.